twake/backend/node/src/services/documents/entities/file-version.ts

```diff
@@ -4,7 +4,7 @@ import { Column, Entity } from "../../../core/platform/services/database/service
 export const TYPE = "drive_file_versions";
 
 @Entity(TYPE, {
-  primaryKey: [["file_id"], "id"],
+  primaryKey: [["id"], "file_id"],
   type: TYPE,
 })
 export class FileVersion {
```

Comment:
Please rename "id" to "drive_file_id", I think it will be more clear wdyt ? @rezk2ll

Reply:
I think the `file_id` is supposed to be `drive_file_id` and the `id` is just the id of the version

Reply:
In this case there is still an issue because when retrieving the list of versions you do it per "drive_file_id", it is like a folder structure: `/{drive_file_id}/{version_id}` so you can get all the versions from a drive_file_id or a single version but then need both drive_file_id and version_id.
In the change you made the directory structure is `/{id}/{file_id}` so file_id can't be equivalent to the drive_file_id right ?

Reply:
@rezk2ll I let you see with @MontaGhanmy